Code/GraphMol/RascalMCES/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+
+rdkit_library(RascalMCES
+        RascalMCES.cpp RascalCluster.cpp RascalButinaCluster.cpp
+        lap_a_la_scipy.cpp PartitionSet.cpp RascalResult.cpp
+        LINK_LIBRARIES SmilesParse FileParsers ChemTransforms SubstructMatch GraphMol)
+target_compile_definitions(RascalMCES PRIVATE RDKIT_RASCALMCES_BUILD)
+
+rdkit_headers(RascalMCES.h RascalOptions.h RascalClusterOptions.h RascalResult.h
+        DEST GraphMol/RascalMCES)
+
+rdkit_test(testRascalMCES mces_catch.cpp LINK_LIBRARIES RascalMCES)
```

Comment:
it's easier to use tests with catch2 if you make this an `rdkit_catch_test`. Then you don't need to have such gross includes in the files

Reply:
It will also allow you to ditch the `#define CATCH_CONFIG_MAIN` from the test files, which will *dramatically* improve compile times